ptx/sec_numerical_integration.ptx

```diff
@@ -566,7 +624,14 @@
       <caption>Approximating <m>\int_0^1 e^{-x^2}\, dx</m> using 5 trapezoids of equal widths</caption>
         <!-- START figures/fig_num3a.tex -->
       <image xml:id="img_num3a" width="47%">
-        <description></description>
+      <shortdescription>approximating the integral of e^{-x^2} from 0 to 1 using five trapezoids</shortdescription>
+        <description>
+        <p> The graph of <m>e^-{x^2}</m> starts from <m>(0,1)</m> and slopes downwards to ht right in the positive x direction. We consider the are under the curve from 
```

Comment:
`e^{-x^2}`, not `e^-{x^2}`

not sure what "ht" should be. "the", probably?

Math tags on x